packages/lab/src/layout/LayerLayout/LayerLayout.css

```diff
@@ -0,0 +1,112 @@
+.uitkLayerLayout {
+  --layer-border-color: var(--uitkLayerLayout-border-color, var(--uitk-container-border-color-medium));
+  --layer-box-shadow: var(--uitkLayerLayout-box-shadow, var(--uitk-overlayable-shadow-modal));
+  --layer-background-color: var(--uitkLayerLayout-background-color, var(--uitk-container-background-medium));
+  --layer-border-width: var(--uitkLayerLayout-border-color, var(--uitk-container-border-width));
+  --layer-padding: var(--uitkLayerLayout-padding, calc(3 * var(--uitk-size-unit)));
+  --layer-border-style: var(--uitkLayerLayout-border-style, var(--uitk-container-border-style));
+  --layer-margin: var(--uitkLayerLayout-margin, calc(3 * var(--uitk-size-unit)));
+  --uitk-animation-duration: 0.2s;
```

Comment:
We should add this into the foundation as some sort of state/variant instead of override it here